plugin/metadata/README.md

```diff
@@ -0,0 +1,29 @@
+# metadata
+
+## Name
+
+*metadata* - enables a metadata.
+
+## Description
+
+By enabling *metadata* any plugin that implements [metadata.metadater interface](https://godoc.org/github.com/coredns/coredns/plugin/metadata#Metadater) will be called and will be able to add it's own Metadata to context. Metadata is available for all plugins via Context.
```

Comment:
The metadata collected will be available for all plugins handler, via the Context parameter provided in the ServeDNS function